### PR TITLE
feat(virtual-background): Resize and drag and drop on virtual backgro…

### PR DESCRIPTION
**Pull request by tudor-phd**
…und.

### CODE REVIEW

File: react/features/stream-effects/virtual-background/JitsiStreamBackgroundEffect.js [hunk 1]
```diff
@@ -140,16 +140,34 @@ export default class JitsiStreamBackgroundEffect {
...
-                backgroundType === VIRTUAL_BACKGROUND_TYPE.IMAGE
-                    ? this._virtualImage : this._virtualVideo,
+                this._virtualImage,
                 0,
                 0,
                 this._outputCanvasElement.width,
                 this._outputCanvasElement.height
             );
+        }
+        if (backgroundType === VIRTUAL_BACKGROUND_TYPE.DESKTOP_SHARE) {
```
Comment (saghul) [inline react/features/stream-effects/virtual-background/JitsiStreamBackgroundEffect.js:153]: use an else-if here please

**Comment by tudor-phd** [comment]
Known issues: 
Screenshot 2021-09-07 at 15 18 26
Screenshot 2021-09-07 at 15 13 18

**Comment by saghul** [comment]
@tudor-phd Is this ready for review? If so, can you mark it as not WIP?

**Comment by tudor-phd** [comment]
> @tudor-phd Is this ready for review? If so, can you mark it as not WIP?

Changed to 'open PR'. Unfortunately, known issues are still available.

**Comment by tudor-phd** [comment]
Fixed this issue on last commits: https://user-images.githubusercontent.com/18324739/132344023-1fd5cee8-033e-4bd9-babb-3b25e06bf66e.png

**Comment by tudor-phd** [comment]
Fixed this issue on last commits: https://user-images.githubusercontent.com/18324739/132344003-98b1d29c-0139-489c-a24a-50f5debe71c7.png

**Comment by tudor-phd** [comment]
"Going back to none is broken" - fixed

**Comment by saghul** [comment]
Dropping this for now, as discussed.

**Pull request closed without merging**